<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputOutput</class>
 <widget class="QWidget" name="InputOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_Btn" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <widget class="QPushButton" name="Btn_Filter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>133</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>过滤器</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_0">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>0</y>
      <width>133</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_1">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>0</y>
      <width>133</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_refresh">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>0</y>
      <width>133</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_View">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>0</y>
      <width>135</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>视图</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_simulation">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>0</y>
      <width>133</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>仿真</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>800</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentIOType">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>当前信号类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>活动过滤器:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filterShow">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ChooseLayout">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>选择布局</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ChooseIO">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>从列表中选择一个I/O信号。</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Layout">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>默认</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>标签</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
